torch/_torch_docs.py

```diff
@@ -6715,15 +6717,23 @@ def merge_dicts(*dicts):
         from the QR decomposition.
 
 Args:
-    input (Tensor): tensor of shape `(*, m, k)` where `*` is zero or more batch dimensions.
-    tau (Tensor): tensor of shape `(*, min(m, k))` where `*` is zero or more batch dimensions.
+    input (Tensor): tensor of shape `(*, mn, k)` where `*` is zero or more batch dimensions.
+    tau (Tensor): tensor of shape `(*, min(mn, k))` where `*` is zero or more batch dimensions.
     other (Tensor): tensor of shape `(*, m, n)` where `*` is zero or more batch dimensions.
-    left (bool): controls the order of multiplication
+    left (bool): controls the order of multiplication.
     transpose (bool): controls whether the matrix `Q` is conjugate transposed or not.
 
 Keyword args:
     out (Tensor, optional): the output Tensor. Ignored if `None`. Default: `None`.
 
+Raises:
+    RuntimeError:
+        if :attr:`input` and :attr:`other` have incompatible shapes for the matrix multiplication.
```

Comment:
We are not reporting the size errors in this section, as basically every function has some "size compatibility requirements" and it'd mean too much repetition.

Reply:
Okay, I'll remove this section then and also the note about tau as it's also a size incompatibility issue.